src/Controls/src/Core/Internals/InvalidationTriggerFlags.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+
+namespace Microsoft.Maui.Controls.Internals;
+
+[Flags]
+internal enum InvalidationTriggerFlags : ushort
```

Comment:
`InvalidationTrigger` is a public API but at the same time it says

> For internal use by platform renderers.

So I wonder if I could simply change that to:

```
[Flags]
public enum InvalidationTrigger : ushort
{
	None = 0,
	BatchingBindingContext = 1 << 0,
    // leaving empty space here in case we want to add other types of batching
 	NotifyOnChildMeasureInvalidated = 1 << 4,
	HorizontalOptionsChanged = 1 << 5,
	VerticalOptionsChanged = 1 << 6,
	MarginChanged = 1 << 7,
	SizeRequestChanged = 1 << 8,
	MeasureChanged = 1 << 9,
	RendererReady = 1 << 10,
	Undefined = 1 << 11,
}
```